dora/core/client/fs/src/main/java/alluxio/client/file/cache/NoExceptionCacheManager.java

```diff
@@ -47,7 +47,7 @@ public void commitFile(String fileId) {
     try {
       mCacheManager.commitFile(fileId);
     } catch (Exception e) {
-      LOG.error("Failed to commit file {}", fileId);
+      LOG.error("Failed to commit file {}", fileId, e);
```

Comment:
this adds the exception to the log, just like what all other methods here are doing